prov/efa/src/rxr/rxr_init.c

```diff
@@ -690,7 +693,9 @@ EFA_INI
 	fi_param_define(&rxr_prov, "shm_cq_read_size", FI_PARAM_SIZE_T,
 			"Set the number of SHM completion entries to read for one loop for one iteration of the progress engine. (Default: 50)");
 	fi_param_define(&rxr_prov, "inter_max_medium_message_size", FI_PARAM_INT,
-			"The maximal message size for inter EFA medium message protocol, messages whose size is larger than this value will be sent either by read message protocol (depend on firmware support), or long message protocol (Default 131072).");
+			"The maximal message size for inter EFA medium message protocol, messages whose size is larger than this value will be sent either by read message protocol (depend on firmware support), or long message protocol (Default 65536).");
+	fi_param_define(&rxr_prov, "inter_max_long_message_size", FI_PARAM_INT,
+			"The maximal message size for inter EFA long message protocol, messages whose size is larger than this value will be sent either by read message protocol (if firmware support) (Default 131072).");
 	fi_param_define(&rxr_prov, "inter_max_emulated_read_size", FI_PARAM_INT,
```

Comment:
Do you think there's value in keeping the emulated read/write message size parameters instead of using 'inter_max_long_message_size'?

Reply:
I prefer keep them separate because they are used for different protocol.
However, the variable "max_emulated_write" is inaccurate, should be "max_long_write" because write is always emulated. I will fix it in new revision.